alf/environments/suite_gym.py

```diff
@@ -76,6 +76,7 @@ def wrap_env(gym_env,
              max_episode_steps=0,
              gym_env_wrappers=(),
              time_limit_wrapper=alf_wrappers.TimeLimit,
+             normalize_action=True,
```

Comment:
Might be better to make the default value to be False, so that the default behavior is the same as the original env?

Reply:
What do you mean by default behavior?

Reply:
> Might be better to make the default value to be False, so that the default behavior is the same as the original env?

What do you mean by default behavior?